object/range.go

```diff
@@ -5,47 +5,61 @@ import (
 )
 
 // Range represents v2-compatible object payload range.
```

Comment:
```suggestion
// Range represents payload range of the NeoFS object.
```

Reply:
why? thought all public types should have such info

Reply:
About V2? We say about that below.